### DISCUSSION TITLE
minio pod CrashLoopBackOff , storage is nfs pvc

### DISCUSSION THREAD

**Discussion by linpanru** [minio/minio#16787] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/56077231/224018647-03085cb2-a6fa-40a7-9b88-45cb27d8c8d6.png)

i use nfs pvc for  minio  storage,  first of all, all is ready ,but one day ,nfs-server is down ,then i change to another nfs-server(the data is backdata), then ,i restart minio pod ,they are CrashLoopBackOff

**Comment by vadmeste** [comment, marked as answer]
@linpanru this looks like a simple permission issue. You need to make sure that MinIO binary is allowed to read/write in the NFS mount. (NFS by the way is not recommended use case here)

**Reply by linpanru** [reply]
> @linpanru this looks like a simple permission issue. You need to make sure that MinIO binary is allowed to read/write in the NFS mount. (NFS by the way is not recommended use case here)
so  What do you recommend?

**Reply by harshavardhana** [reply]
Use disk filesystems such as XFS with locally attached PV.

**Reply by linpanru** [reply]
> Use disk filesystems such as XFS with locally attached PV.

thanks for your help, i will try
